### PR TITLE
Map Display

### PR DESCRIPTION
**Pull request by 18lz**
Realized the display of the map on the homepage and the independent display after zooming in.
WechatIMG3
WechatIMG2

### CODE REVIEW

File: app/src/main/java/org/soundscape/screens/MapPage.kt [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-                modifier = Modifier.padding(8.dp),
-                text = "Placeholder",
-                color = Color.White
-            )
+            Box(modifier = Modifier.fillMaxSize()) {
+                SoundscapeMap(
+                    modifier = Modifier.fillMaxSize(),
+                    onInitMapView = {
+                        it.controller.setZoom(zoomLevel)
+                        it.controller.setCenter(GeoPoint(51.524273,-0.1343282))
```
Comment (dragonfist453) [inline app/src/main/java/org/soundscape/screens/MapPage.kt:26]: Would be good to have a marker here as well showing the point

File: app/src/main/java/org/soundscape/components/MapCard.kt [hunk 2]
```diff
@@ -102,6 +104,16 @@ fun MapCard(navController: NavHostController) {
                     Box(
                         modifier = Modifier.fillMaxSize()
                     ) {
+                        var zoomLevel = 17.0
+                        SoundscapeMap(
+                            modifier = Modifier.fillMaxSize(),
+                            onInitMapView = {
+                                it.controller.setZoom(zoomLevel)
+                                // TODO change to variable for future map display
```
Comment (dragonfist453) [inline app/src/main/java/org/soundscape/components/MapCard.kt:112]: This comment is not required here

File: app/src/main/java/org/soundscape/components/MapCard.kt [hunk 3]
```diff
@@ -102,6 +104,16 @@ fun MapCard(navController: NavHostController) {
...
+                                it.controller.setCenter(GeoPoint(51.524273,-0.1343282))
```
Comment (dragonfist453) [inline app/src/main/java/org/soundscape/components/MapCard.kt:113]: Would be nice with a Marker, like for MapPage

**Review by dragonfist453 — APPROVED** [review]
The comments are minor and this can be merged anyway

File: app/src/main/java/org/soundscape/components/MapCard.kt (hunk 3 above)
Comment (dragonfist453) [reply app/src/main/java/org/soundscape/components/MapCard.kt:113]: But we can do this after we move to Google Maps

File: app/src/main/java/org/soundscape/screens/MapPage.kt (hunk 1 above)
Comment (dragonfist453) [reply app/src/main/java/org/soundscape/screens/MapPage.kt:26]: nvm, can do after we move to Google maps

**Review by nextedoff — APPROVED** [review]
Looks good to me.

**Pull request merged**